src/batch/src/spill/spill_op.rs

```diff
@@ -39,25 +39,41 @@ const RW_MANAGED_SPILL_DIR: &str = "/rw_batch_spill/";
 const DEFAULT_IO_BUFFER_SIZE: usize = 256 * 1024;
 const DEFAULT_IO_CONCURRENT_TASK: usize = 8;
 
+#[derive(Clone)]
+pub enum SpillType {
+    Disk,
+    Memory,
```

Comment:
```suggestion
    /// Only for testing purpose
    Memory,
```

Otherwise it's quite confusing...